src/cli.js

```diff
@@ -26,14 +26,14 @@
 
   let argv, activeConfig;
   let args = process.argv;
-  let commandParam = args[2];
+  let commandParam = args[3];
```

Comment:
This may nit be always the case
What if i have a single word command
davos upload
davos meta upload
will this work as well ?